docker-compose.test.yml

```
@@ -0,0 +1,30 @@
+services:
+  prod:
+    healthcheck:
+      test: [ "CMD", "curl", "-f", "http://localhost:3001/api/health" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)